pom.xml

```diff
@@ -38,7 +38,7 @@
         <!-- Dependencies -->
         <slf4j.version>1.7.7</slf4j.version>
         <protobuf.version>3.6.1</protobuf.version>
-        <grpc.version>1.19.0</grpc.version>
+        <onos.grpc.version>1.22.1</onos.grpc.version>
```

Comment:
We should not have an ONOS property in the Atomix client repo. Just change the version but not the property name.

Reply:
done